scripts/migrations/migration.003.data_packages.py

```diff
@@ -104,7 +104,7 @@ def update_column_type_metadata(bucket: str):
             output.setdefault(study, {})
             output[study].setdefault(data_package, {})
             output[study][data_package].setdefault(version, {})
-            output[study][data_package][version]["column_types_format_version"] = 2
+            output[study][data_package][version]["column_types_format_version"] = "2"
```

Comment:
i realize this is historical but i'm just trying to squash all of these and be done with it.